components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/executors/ApiPublisherExecutor.java

```diff
@@ -110,6 +105,7 @@ private static APIStatus getApiStatus(String status) {
         for (APIStatus aStatus : APIStatus.values()) {
```

Comment:
Instead of the this  getApiStatus private method, you can use getApiStatus public method available in the APIUtil class. So you can avoid the code redundancy.